### PR TITLE
[LOOP-29] Initial Tidepool Service plugin

### PR DESCRIPTION
**Pull request by darinkrauss**
- Initial, placeholder Tidepool Service plugin

### CODE REVIEW

File: TidepoolServiceKitUI/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/* Button title to delete a service */
+"Delete Service" = "Delete Service";
```
Comment (nhamming) [inline TidepoolServiceKitUI/de.lproj/Localizable.strings]: I assume there are all just placeholders to remind which languages are targeted? Or should these `.strings` files be removed until translation is ready?

**Review by nhamming — APPROVED** [review]
LGTM

File: TidepoolServiceKitUI/de.lproj/Localizable.strings (hunk 1 above)
Comment (darinkrauss) [reply TidepoolServiceKitUI/de.lproj/Localizable.strings]: Yes, placeholders for later translation.

**Comment by darinkrauss** [comment]
@ps2 Not sure if you want to chime in here. This is just a basic empty service plugin. There is no actual functionality in here (other than a basic service plugin), but it is the shell for the upcoming Tidepool synchronization implementation.

**Review by ps2 — APPROVED** [review]
LGTM

**Pull request merged**